src/reports/controllers/reportsController.js

```diff
@@ -3,6 +3,8 @@
 const aggregateReportGenerator = require('../models/aggregateReportGenerator');
 const reports = require('../models/reportsManager');
 const stats = require('../models/statsManager');
+const { re } = require('mathjs');
```

Comment:
both line 6 and 7 not in use

Reply:
we have two openapi specs in this project
1. predator/docs/openapi3.yaml - for validation
2. predator/docs/devguide/docs/swagger-docs.yaml

need to update them both with new api